src/puptoo/process/profile.py

```diff
@@ -258,11 +258,16 @@ def system_profile(
                 profile["sap_sids"] = sorted(list(sids))
                 inst = instances[0]
                 profile["sap_instance_number"] = sap[inst].number
-
+            sap_object = {
+                "sap_system": profile.get("sap_system"),
+                "sap_sids": profile.get("sap_sids"),
+                "sap_instance_number": profile.get("sap_instance_number")
```

Comment:
To stay consistent with other object naming conventions, we dropped the duplicate `sap_` where possible, so keep `sap_system`, but then it should be just `sids`, `instance_number`, and `version`.  See our [schema here](https://github.com/RedHatInsights/inventory-schemas/blob/master/schemas/system_profile/v1.yaml#L426-L453).

Although I don't see `sap_version` among the existing fields; did puptoo remove this recently?  We still have it in our [schema here](https://github.com/RedHatInsights/inventory-schemas/blob/master/schemas/system_profile/v1.yaml#L472-L477).

Reply:
Fixed! Added SAP `version` as well. Thanks!